### PR TITLE
Domain balance lambdas fixes

### PR DESCRIPTION
**Pull request by mmioana**
## Description

Fix: Add missing imports and edge-cases handling
Fix: Update lambdas Timeout and MemorySize

After discussing with @bogdan-1337 it makes sense to refactor the coin gecko variables to be retrieved from ssm, so I added a few more refactoring commits

Refactor: Add coinGeckoApiUrl and coinGeckoApiKey to ssm lambda layer
Refactor: Use coingecko api url and key from ssm and rename env vars config
Refactor: Move and rename ExchangeRatesFactory to ExchangeRatesService

### CODE REVIEW

**Review by rdig — DISMISSED** [review]
All good here.

The only "eyebrow raising" thing here would be the `1000` limit, which AWS limits to a max data size of 1mb per request, meaning you might get less than 1000 entries at times.

But since this is all paginated, it's not really a issue, at worst the query ends up doing more round trips

File: amplify/backend/function/fetchDomainBalance/src/config/exchangeRates.js [hunk 1]
```diff
@@ -54,7 +56,8 @@ const ExchangeRatesFactory = (() => {
   };
 
   const getApiTokenIdFromAddress = async ({ tokenAddress, chainId }) => {
-    const { mappings } = await CoinGeckoConfig.getConfig();
+    const { mappings, DEFAULT_NETWORK_TOKEN } =
```
Comment (rumzledz) [inline amplify/backend/function/fetchDomainBalance/src/config/exchangeRates.js:59]: 👌

**Review by rumzledz — APPROVED** [review]
Nothing stands out to me @mmioana ! Looking forward to your fixes once deployed 🙌

**Review by bogdan-1337 — APPROVED** [review]

**Review by iamsamgibbs — APPROVED** [review]
All the changes here look sensible!

**Review by rumzledz — APPROVED** [review]
Re-approving ✅

**Pull request merged**